internal/server/proto/server.proto

```diff
@@ -801,6 +815,12 @@ message Ref {
   message DeclaredResource {
     string name = 1;
   }
+
+  // OndemandRunner references an OndemandRunner value to be used for contextual
+  // operation.
+  message OndemandRunner {
```

Comment:
Similarly, this is referencing the ODR _config_ right, and not a running runner itself? As a reader of these protos, a `Config` suffix would clarify that for me.